### PR TITLE
Reorg docs a bit

### PR DESCRIPTION
**Pull request by advayDev1**
- Trim the README down to the minimum required to start
- Move the longer sections to FAQ, on Xcode, directories, deps, and build time
- Give instructions on bugs

### CODE REVIEW

**Comment by advayDev1** [comment]
@brunobowden - ptal.  I think the README had gotten unwieldly.  We need new users to read all the way to the end.  So I kept the summary, how to install, how to run, and how to file problems.  I moved the build speed, folders, and Xcode recommendations to FAQ.

**Comment by advayDev1** [comment]
I also added a ToC to the FAQ (semi-auto-generated) and added a pointer to dependencies to README.  Looks like the current most common question is how to rely on 3rd party jars.

File: README.md [hunk 1]
```diff
@@ -46,7 +46,10 @@ j2objcConfig {
 }
 ```
 
-Info on additional `j2objcConfig` settings are in [J2objcConfig.groovy](https://github.com/j2objc-contrib/j2objc-gradle/blob/master/src/main/groovy/com/github/j2objccontrib/j2objcgradle/J2objcConfig.groovy#L30).
+Info on additional `j2objcConfig` settings are in [J2objcConfig.groovy](https://github.com/j2objc-contrib/j2objc-gradle/blob/master/src/main/groovy/com/github/j2objccontrib/j2objcgradle/J2objcConfig.groovy#L30).  If your `shared` project depends on
```
Comment (brunobowden) [inline README.md]: Start new line for **If your `shared` project depends on...**

File: README.md [hunk 2]
```diff
@@ -99,106 +102,28 @@ For a complete build, run both:
...
-    j2objcXcode       - Xcode updated with libraries, headers & resources (uses CocoaPods)
-
-Running the `build` task from the Gradle Java plugin will automatically run the j2objcBuild command
-and all the previous tasks (which it depends on). Only the `j2objcXcode` task needs to be manually
-run. Note that you can use the Gradle shorthand of `$ gradlew jA` to do the `j2objcAssemble` task.
-The other shorthand expressions are `jCF, jTr, jA, jTe, jB and jX`.
+If you are not comfortable sharing these, the community may not be able to help as much.
 
+Mozilla's [Bug writing guidelines](https://developer.mozilla.org/en-US/docs/Mozilla/QA/Bug_writing_guidelines)
+may be helpful.  Having public, focused, and actionable Issues
```
Comment (brunobowden) [inline README.md]: Single space after period is our convention.

File: README.md [hunk 3]
```diff
@@ -99,106 +102,28 @@ For a complete build, run both:
...
+helps the maximum number of users and also lets the maximum number of people help you.
+Please do not email the authors directly.
 
 ### FAQ
 
-See [FAQ.md](FAQ.md).
+Please [FAQ.md](FAQ.md).
```
Comment (brunobowden) [inline README.md]: => Please see

**Comment by brunobowden** [comment]
LGTM - The TOC instructions are very useful

File: README.md (hunk 1 above)
Comment (advayDev1) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (advayDev1) [inline README.md]: done

File: README.md (hunk 3 above)
Comment (advayDev1) [inline README.md]: done

**Comment by advayDev1** [comment]
little bit of jQuery goes a long way ;)

**Pull request merged**